web/client/components/data/download/ExportDataResultsComponent.jsx

```diff
@@ -50,23 +51,28 @@ const ExportDataResultsComponent = ({
                     <DefaultInfoBubbleInner {...infoBubbleMessage} />
                 </InfoBubble>
             </div> : null}
-            <Dialog
-                id="mapstore-export-data-results"
-                style={{display: active ? "block" : "none"}}
-                draggable={false}
-                modal>
-                <span role="header">
-                    <span className="about-panel-title"><Message msgId="exportDataResults.title"/></span>
-                    <button onClick={() => onToggle()} className="settings-panel-close close"><Glyphicon glyph="1-close"/></button>
-                </span>
-                <div role="body">
-                    <ExportDataResults
-                        loading={checkingExportDataEntries}
-                        results={results}
-                        currentLocale={currentLocale}
-                        onRemoveResult={onRemoveResult}/>
-                </div>
-            </Dialog>
+            {/* Portal must be contained in a valid node tag not in <></> */}
+            <div>
+                {active && <Portal>
+                    <Dialog
+                        id="mapstore-export-data-results"
+                        style={{display: active ? "block" : "none"}}
```

Comment:
We can remove style display property manipulation, since we are conditionally rendering the component `{active && <Portal>`